src/main/java/seedu/address/logic/commands/slot/FindSlotCommand.java

```diff
@@ -0,0 +1,44 @@
+package seedu.address.logic.commands.slot;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.core.PshMessages;
+import seedu.address.logic.commands.general.CommandResult;
+import seedu.address.logic.commands.general.PshCommand;
+import seedu.address.model.PshModel;
+import seedu.address.model.slot.SlotContainsKeywordsPredicate;
+
+/**
+ * Finds and lists all slots in the schedule whose contents contains any of the argument keywords.
+ * Keyword matching is case insensitive.
+ */
+public class FindSlotCommand extends PshCommand {
```

Comment:
the behaviour here is entirely different from that stated in the [user guide](https://ay1920s2-cs2103-w15-4.github.io/main/UserGuide.html#finding-slots-code-findslots-code), pls read that again and ensure that it matches closely